src/components/AddRepoModal/AddRepoModal.tsx

```diff
@@ -0,0 +1,26 @@
+import { requireAuth } from '@/lib/auth/requireAuth';
+import { SearchComponent } from './SearchComponent';
+
+export default async function AddRepoModal() {
```

Comment:
I don't think this component has to be a sever component (`async`)
```suggestion
export default function AddRepoModal() {
```